package.json

```diff
@@ -37,7 +37,7 @@
     "@babel/eslint-parser": "^7.21.3",
     "@babel/plugin-proposal-decorators": "^7.21.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.0.1",
+    "@ember/string": "^4.0.0",
```

Comment:
`@ember/string` @ v4 _can't_ support ember-source < 3.28 

So... it might be impossible to use `@ember/string` @ v4 until we do a modern version of ember-inflector (which @mansona already has a PR for)